io.github.woelper.Oculante.json

```diff
@@ -0,0 +1,49 @@
+{
+  "app-id": "io.github.woelper.Oculante",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable"
+  ],
+  "command": "oculante",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--device=dri",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--filesystem=host"
```

Comment:
this is a broad filesystem permission.

Reply:
May I ask why this should be changed? Other projects under the same genre have this permission allowed, most notably org.gnome.Loupe.

Reply:
Has since been changed to home